scripts/docker-compose-mysql/docker-compose-horicromtal.yml

```diff
@@ -0,0 +1,43 @@
+# Startup:
+# ```
+# CROMWELL_TAG=develop
+# docker-compose -f scripts/docker-compose-mysql/docker-compose-horicromtal.yml up
+# ```
+
+# Cleanup:
+# ```
+# docker-compose -f scripts/docker-compose-mysql/docker-compose-horicromtal.yml down -v
+# ```
+#
+# Set CROMWELL_BUILD_RESOURCES_DIRECTORY explicitly so fully qualified paths to required credential files are rendered
+# into the CI config files.
+version: '2.3'
+services:
+  cromwell_lord:
+    image: "broadinstitute/cromwell:${CROMWELL_TAG}"
+    network_mode: host
+    working_dir: /cromwell-working-dir
+    volumes:
+      - ${CROMWELL_BUILD_ROOT_DIRECTORY}:${CROMWELL_BUILD_ROOT_DIRECTORY}
+    command: ["server"]
+    environment:
+      - JAVA_OPTS=-Dconfig.file=${CROMWELL_BUILD_RESOURCES_DIRECTORY}/${CROMWELL_CONFIG} -Dwebservice.port=8000 -Dsystem.max-workflow-launch-count=1 -Dsystem.new-workflow-poll-rate=10 -Dsystem.max-concurrent-workflows=30
```

Comment:
So is this analogous to our `backend` Cromwell from prod?

Reply:
This whole setup is a bit simplified and different from what we'll be doing in real environments. This "lord" is like a backend + summarizer, the "soldier" is like a backend + frontend. i.e. both of them run workflows, only the soldier talks to Centaur, only the lord summarizes. I'll add comments since none of this is very clear. 🙂

Reply:
...and also I'll try using better names 😉